dashboard/app/models/user.rb

```diff
@@ -2429,7 +2429,7 @@ def school_info_school
 
   def show_census_teacher_banner?
     # Must have an NCES school to show the banner
-    users_school = try(:school_info).try(:school)
+    users_school = school_info_school
```

Comment:
Should [this line](https://github.com/code-dot-org/code-dot-org/blob/ff74a3a7c4e44dd5ce88c2e56f15e48fe9e38eef/dashboard/app/controllers/home_controller.rb#L205) be updated to match? I know they're almost the same so it may be unnecessary

Reply:
hmm... actually maybe we should use the `current_school` query here instead? `last_complete()&.school` will only ever return an nces school, whereas a user may have entered an unlisted school or one out of the country 🤔

Reply:
oh nope! the comment clearly says it needs to be an nces school to show the banner 😂

Reply:
updated to match now 🙌